### ISSUE TITLE
Trying to build an application using msdk-all with gradle fails

### ISSUE THREAD

**Issue by bjonnh** [msdk/msdk#279]
Using: implementation("io.github.msdk:msdk-all:0.0.27")

 Could not find uk.ac.ebi.pride:jmztab-modular-model:3.0.7.
edu.ucdavis.fiehnlab.splash:core:1.8…

Where are all of these?

**Comment by tomas-pluskal** [comment]
We use maven, not gradle. 

The artifacts should be available in the repositories that are referenced in the corresponding pom.xml files, e.g.: https://github.com/msdk/msdk/blob/master/msdk-io-mztab/pom.xml#L20

**Comment by nilshoffmann** [comment]
Hi,
it seems that the EBI Repository is redirecting to https from the http address configured in the pom.


However, the resolution works for me. Unfortunately, there is a SNAPSHOT dependency that leads 

to a compilation error at the moment for both master and v0.0.19:


[ERROR] Failed to execute goal on project msdk-id-sirius: Could not resolve dependencies for project io.github.msdk:msdk-id-sirius:jar:0.0.28-SNAPSHOT: Failed to collect dependencies at de.unijena.bioinf.ms:gibbs_sampling:jar:4.0.1 -> de.unijena.bioinf.ms:sirius_api:jar:4.0.1 -> de.unijena.bioinf.ms:fragmentation_tree_construction:jar:4.0.1 -> de.unijena.bioinf.ms:Recalibration:jar:1.0 -> de.unijena.bioinf.ms:chemistry_base:jar:3.1.4-SNAPSHOT:


If I update to a newer release version of gibbs_sampling, (4.4.8: https://bio.informatik.uni-jena.de/repository/libs-oss/de/unijena/bioinf/ms/gibbs_sampling/4.4.8/)
this gives me another SNAPSHOT dependency on jjobs-core: https://bio.informatik.uni-jena.de/repository/libs-oss/de/unijena/bioinf/jjobs-core/0.9.17-SNAPSHOT/


If I add the SNAPSHOT repository explicitly, the compilation progresses further, but stops at 



[ERROR] Failed to execute goal on project msdk-id-sirius: Could not resolve dependencies for project io.github.msdk:msdk-id-sirius:jar:0.0.28-SNAPSHOT: Failed to collect dependencies at de.unijena.bioinf.ms:gibbs_sampling:jar:4.4.8 -> de.unijena.bioinf.ms:sirius_api:jar:4.4.8 -> de.unijena.bioinf.ms:sirius_isotopes:jar:4.4.8 -> de.unijena.bioinf.ms:fragmentation_tree_construction:jar:4.4.8 -> de.unijena.bioinf.ms:Recalibration:jar:1.0 -> de.unijena.bioinf.ms:chemistry_base:jar:3.1.4-SNAPSHOT



Best wishes,
Nils

**Comment by bjonnh** [comment]
Yes you use maven, but I use gradle to build my program, that's what I meant, sorry I updated the title.

**Comment by bjonnh** [comment]
Yes some of de.unijena.bioinf.ms  are failing as well…

**Comment by albertogilf** [comment]
Hi, I have the same issue: 

I had problems with maven 3.8.1 to resolve http dependencies when trying to use the library (fiehn lab). Basically maven does not resolve dependencies from http repos in newer versions. I have downgrade the maven version to maven 3.6.1 and I now the problem is that the dependencies of de.unijena.bioinf are not found :(. 
I can still use the package and work with it, but I wonder if that dependencies are necessary for any feature.
